src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-baseline-alert-conditions.mdx

```diff
@@ -59,6 +59,20 @@ Here are some tips for setting baseline thresholds:
 * For NRQL alerts, see the [allowed types of NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types).
 * If the alert condition applies to multiple apps, you can select a choice from the dropdown above the chart to use different metrics. (Not applicable for NRQL alert conditions.)
 
+## Faceted baseline conditions [#baseline-facets]
+
+![A screenshot of an example baseline condition with facets.](./images/faceted-baseline-condition-query.png "An example of a baseline condition with facets.")
+
+<figcaption>NRQL alerts conditions support faceted baselines. In the chart legend, click a time series to scope it to a single facet.</figcaption>
+
+Once you've input a faceted NRQL query for a baseline condition, you can scope your results to a single baseline. 
+
+![A screenshot of a baseline scope to a single time series facet.](./images/baseline-one-facet.png "An example of a baseline condition scoped to a single time series facet.")
```

Comment:
The important link is in the bottom-left corner of this image. How about chopping it in half so it's easier to read?